### PR TITLE
Bump nodejs-react-{redux,numeric-input}

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ehelms** [comment]
[test]

File: nodejs-react-numeric-input/nodejs-react-numeric-input.spec [hunk 1]
```diff
@@ -1,48 +1,40 @@
...
 %prep
 %setup -q -n package
 
 %build
-%nodejs_symlink_deps --build
 
 %install
 mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
-cp -pfr .eslintignore .eslintrc .flowconfig .npmignore .travis.yml CHANGELOG.md LICENSE README.md __tests__ build_config dist examples index.js karma.conf.js package.json src %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr CHANGELOG.md LICENSE README.md __tests__ build_config dist docs index.js karma.conf.js nightwatch.conf.js nightwatch.json package.json selenium-download.js src tests_e2e %{buildroot}%{nodejs_sitelib}/%{npm_name}
```
Comment (ekohl) [inline nodejs-react-numeric-input/nodejs-react-numeric-input.spec]: Should I trim out some files here like nightwatch.conf and test_e2e?

Comment (ehelms) [reply nodejs-react-numeric-input/nodejs-react-numeric-input.spec]: If they aren't important for runtime.

Comment (ekohl) [reply nodejs-react-numeric-input/nodejs-react-numeric-input.spec]: I trimmed down to what I think is the minimal required set

Comment (ekohl) [reply nodejs-react-numeric-input/nodejs-react-numeric-input.spec]: I'm still doubting index.js, it is in package.json as main so I'll leave it in for now.

**Comment by ehelms** [comment]
ACK

**Pull request merged**